pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -62,17 +62,17 @@ self: super: {
 
   # The Hackage tarball is purposefully broken, because it's not intended to be, like, useful.
   # https://git-annex.branchable.com/bugs/bash_completion_file_is_missing_in_the_6.20160527_tarball_on_hackage/
-  git-annex = ((overrideCabal super.git-annex (drv: {
+  git-annex = dontCheck (((overrideCabal super.git-annex (drv: {
```

Comment:
Fair enough I'll try to give it a whack sometime today. The test suite isn't finding the git-annex-shell binary to test with